engine/platform/sdl/in_sdl.c

```diff
@@ -202,10 +198,7 @@ SDLash_JoyInit_New
 */
 static int SDLash_JoyInit_New( int numjoy )
 {
-	int temp, num;
-	int i;
-
-	Con_Reportf( "Joystick: SDL GameController API\n" );
```

Comment:
That was pretty useful line because we support both GameController API and legacy Joystick API.

Reply:
Restored